src/js/jsx/sections/style/Type.jsx

```diff
@@ -63,36 +63,55 @@ define(function (require, exports, module) {
                 return collection.pluck(document.layers.selected, "opacity");
             };
 
+            if (!Immutable.is(this.state.postScriptMap, nextState.postScriptMap)) {
+                if (nextState.postMap) {
+                    // The list of all selectable type faces
+                    var typefaces = nextState.postScriptMap
```

Comment:
Moving this here so it only needs to be recalculated when the `postScriptMap` changes, instead of every time any flux state changes.